docs/advanced-topics/code_coverage.md

```diff
@@ -9,9 +9,10 @@ permalink: /advanced-topics/code-coverage/
 # Code Coverage
 {: .no_toc}
 
-For projects written in C/C++, you can generate code coverage reports using
-Clang source-based code coverage. This page walks you through the basic steps.
-For more details, see [Clang's documentation].
+For projects written in C/C++, Go, or Java and other JVM-based languages,
```

Comment:
nit: can you add rust also. it's also supported now.

Reply:
Done.